api/src/main/java/org/openmrs/module/aijar/metadata/concept/Dictionary.java

```diff
@@ -0,0 +1,157 @@
+package org.openmrs.module.aijar.metadata.concept;
```

Comment:
@moshonk We do not need the concept classes anywhere - you can use the ConceptService instead